.github/workflows/testlint.yml

```diff
@@ -70,25 +70,3 @@ jobs:
     - name: Test with pytest
       run: |
         pytest
-
```

Comment:
https://pypi.org/project/dataclasses/ You can install this conditionally if you wish to continue supporting dataclasses.